### PR TITLE
[orientation] Part 1: Add page orientation predictor

### PR DESCRIPTION
**Pull request by felixdittrich92**
This PR:

- Add page orientation predictor & corresponding tests
- Dedicated as page_orientation_predictor instead of a general orientation_predictor for further handling
- Part 1 of 2
- Part 2 will be the final integration into the OCRPredictor

Any feedback is welcome :hugs:

### CODE REVIEW

**Comment by felixdittrich92** [comment]
setup-python macos issue: https://github.com/actions/setup-python/issues/852

File: doctr/models/classification/mobilenet/pytorch.py [hunk 1]
```diff
@@ -245,3 +246,28 @@ def mobilenet_v3_small_crop_orientation(pretrained: bool = False, **kwargs: Any)
...
     )
+
+
+def mobilenet_v3_small_page_orientation(pretrained: bool = False, **kwargs: Any) -> mobilenetv3.MobileNetV3:
+    """MobileNetV3-Small architecture as described in
+    `"Searching for MobileNetV3",
+    <https://arxiv.org/pdf/1905.02244.pdf>`_.
+    >>> import torch
+    >>> from doctr.models import mobilenet_v3_small_page_orientation
+    >>> model = mobilenet_v3_small_orientation(pretrained=False)
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/pytorch.py:257]:
```suggestion
    >>> model = mobilenet_v3_small_page_orientation(pretrained=False)
```

File: doctr/models/classification/mobilenet/tensorflow.py [hunk 2]
```diff
@@ -414,3 +415,23 @@ def mobilenet_v3_small_crop_orientation(pretrained: bool = False, **kwargs: Any)
...
     return _mobilenet_v3("mobilenet_v3_small_crop_orientation", pretrained, include_top=True, **kwargs)
+
+
+def mobilenet_v3_small_page_orientation(pretrained: bool = False, **kwargs: Any) -> MobileNetV3:
+    """MobileNetV3-Small architecture as described in
+    `"Searching for MobileNetV3",
+    <https://arxiv.org/pdf/1905.02244.pdf>`_.
+    >>> import tensorflow as tf
+    >>> from doctr.models import mobilenet_v3_small_page_orientation
+    >>> model = mobilenet_v3_small_orientation(pretrained=False)
```
Comment (odulcy-mindee) [inline doctr/models/classification/mobilenet/tensorflow.py:426]:
```suggestion
    >>> model = mobilenet_v3_small_page_orientation(pretrained=False)
```

File: tests/tensorflow/test_models_classification_tf.py [hunk 3]
```diff
@@ -102,6 +114,22 @@ def test_crop_orientation_model(mock_text_box):
     assert all(isinstance(pred, float) for pred in classifier([text_box_0, text_box_270, text_box_180, text_box_90])[2])
 
 
+# TODO: uncomment when model is available
```
Comment (odulcy-mindee) [inline tests/tensorflow/test_models_classification_tf.py:117]: You can uncomment this part

**Pull request merged**